semcore/popper/src/index.d.ts

```diff
@@ -92,6 +92,10 @@ export type PopperProps = OutsideClickProps &
      * If enabled, you will need to use setTrigger function from children rendering function to set popper trigger.
      */
     explicitTriggerSet?: boolean;
+    /**
+     * If set, popper will be placed near the place mouse cursor entered the trigger
+     */
+    cursorAnchoring?: 'vertical' | 'horizontal';
```

Comment:
looks like it should be boolean and get type (vertical or horizontal) by placement prop...